nodejs/eks/nodegroup.ts

```diff
@@ -1836,9 +1839,19 @@ Content-Type: text/x-shellscript; charset="us-ascii"
         ? { httpTokens: "required", httpPutResponseHopLimit: 2, httpEndpoint: "enabled" }
         : undefined;
 
+    const blockDeviceMappings = [
+        {
+            deviceName: "/dev/xvda",
```

Comment:
Mental note for myself: This works only for AL2 and AL2023 based AMIs, which is fine right now because we only support AL2. Will need to be taken into account when working on https://github.com/pulumi/pulumi-eks/issues/1195